decidim-core/lib/decidim/form_builder.rb

```diff
@@ -243,6 +243,7 @@ def extract_validations(attribute, options)
       validation_options = {}
       validation_options[:pattern] = "^(.|[\n\r]){#{min_length},#{max_length}}$" if min_length.to_i.positive? || max_length.to_i.positive?
       validation_options[:required] = options[:required] || attribute_required?(attribute)
+      validation_options[:maxlength] ||= max_length if max_length.to_i.positive?
```

Comment:
This is already implemented with the pattern, does it provide a different feedback to the user?